src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayRate = 40 * days;
+  const twentyOff = 20;
+  const fiftyOff = 50;
```

Comment:
Change the names for this two variables - checklist item 5.

Reply:
These are in the tasks description....

Reply:
`twentyOff`:

> Don't hardcode some values that may change in the future in variable names (numbers etc.)

the value twenty may change.